src/stock-operations/batch-no-selector/batch-no-selector.component.tsx

```diff
@@ -102,7 +102,7 @@ const BatchNoSelector = <T,>(props: BatchNoSelectorProps<T>) => {
             }}
             initialSelectedItem={initialSelectedItem}
             itemToString={(s: StockBatchDTO) =>
-              s?.batchNo ? `${s?.batchNo}` : ""
+              s?.batchNo ? `${s?.batchNo} | Qty: ${s?.quantity ?? 0}` : ""
```

Comment:
0 may not always be a good default. We can show a text i.e. unknown for such. thoughts?

Reply:
okay, so I will use an empty string